package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@remoteoss/json-schema-form",
-  "version": "0.7.6-beta.0",
```

Comment:
This will need to be a minor change because it's a BREAKING CHANGE in a beta version.

Reply:
Does the running `npm run release:dev:minor` update this accordingly? This is what I used to update this